src/test/scala/uk/gov/hmrc/perftests/ecl/requests/RegistrationRequests.scala

```diff
@@ -110,17 +116,45 @@ object RegistrationRequests extends Configuration {
 
   val navigateToStubGrsJourneyData: HttpRequestBuilder =
     http("Navigate to /test-only/stub-grs-journey-data")
-      .get(subGrsJourneyDataUrl)
+      .get(stubGrsJourneyDataUrl)
       .check(status.is(200))
       .check(saveCsrfToken)
 
-  def submitStubGrsJourneyData(journeyId: String, businessPartnerId: String): HttpRequestBuilder =
+  val navigateToGrsContinue: HttpRequestBuilder =
+    http("Navigate to /grs-continue/normalmode")
+      .get(redirectLocation("${grsContinueLocation}"))
+      .check(status.is(303))
+
+  def submitStubGrsJourneyData(): HttpRequestBuilder =
     http("Stub GRS Journey Data")
-      .post(subGrsJourneyDataUrl)
+      .post(stubGrsJourneyDataUrl)
       .formParam("csrfToken", "${csrfToken}")
-      .formParam("journeyId", journeyId)
-      .formParam("businessPartnerId", businessPartnerId)
+      .formParam(
+        "grsJourneyDataJson",
+        """{
+            "companyProfile" : {
+              "companyName" : "Test Company Name",
+              "companyNumber" : "01234567",
+              "dateOfIncorporation" : 1196676930000,
```

Comment:
This can be specified as a date now, since this morning we merged an update e.g.

```"dateOfIncorporation" : "2007-12-03"```